packages/flutter_tools/lib/src/compile.dart

```diff
@@ -117,7 +117,8 @@ class PackageUriMapper {
 
     for (String packageName in packageMap.keys) {
       final String prefix = packageMap[packageName].toString();
-      if (fileSystemScheme != null && fileSystemRoots != null && prefix.contains(fileSystemScheme)) {
+      // Only perform a multi-root mapping if there are multiple roots.
+      if (fileSystemScheme != null && fileSystemRoots != null && fileSystemRoots.length > 1 && prefix.contains(fileSystemScheme)) {
```

Comment:
Are there any line-length restrictions in the style guide?

Reply:
Generally try and keep it under 80 - in this case I agree its too far over